modules/gitops-accessing-gitops-monitoring-dashboards.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/monitoring-with-gitops-dashboards.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-accessing-gitops-monitoring-dashboards_{context}"]
+= Accessing {gitops-shortname} monitoring dashboards
+
+The monitoring dashboards are deployed automatically by the Operator. You can access {gitops-shortname} monitoring dashboards from the *Administrator* perspective of the {OCP} web console. 
+
+[NOTE]
+====
+Disabling or changing the content of the dashboards is not supported.
+====
+
+.Prerequisites
+
+* You have access to the {OCP} web console.
+* The {gitops-title} Operator is installed in the default namespace, `openshift-gitops-operator`.
+* The cluster monitoring is enabled on the `openshift-gitops-operator` namespace.
+* You have installed an Argo CD application in your defined namespace, for example, `openshift-gitops`.
```

Comment:
Hello @reginapizza @varshab1210  I added this one line in the prerequisites about deploying an application so I need your approvals again. Everything else is the same. Can you please take a look? Thank you!

Reply:
Hello @adellape can you also please take a look at this one added line so that the peer review is complete again? Thank you!

Reply:
LGTM!